__docs__/wonder-blocks-dropdown/combobox.stories.tsx

```diff
@@ -269,8 +287,13 @@ export const ControlledMultilpleCombobox: Story = {
 
 /**
  * `Combobox` supports autocompletion. This means that the listbox will show
- * options that match the user's input. In this example, we show how this is
- * done by setting the `autoComplete` prop to "list".
+ * options that match the user's input. Note that the search is case-insensitive
+ * and it will match any part of the option item's label. This is useful when
```

Comment:
I think we should be consistent with the search logic, meaning that if we start with filtering anywhere in the label, then we should keep that logic to avoid confusing users.